content/tutorials/python-whirlwind-tour.md

```diff
@@ -225,7 +225,7 @@ The code is executed once for each item in the list, with `x` set to each item i
 [ ] Newspaper
 ```
 
-Unfortunately, this method doesn't tell you the index of the current item. `x` is only a temporary variable, so modifying it has no effect on the list itself (try it). This is where the `enumerate` function comes in. It tells us the index of each value we loop over. An example with numbers:
+Unfortunately, this method doesn't tell you the index of the current item. `x` is only a temporary variable, so modifying it has no effect on the list itself (try it). This is where the `enumerate` function comes in (see the [Calling functions](#calling-functions). It tells us the index of each value we loop over. An example with numbers:
```

Comment:
There seems to be a bracket mismatch here, and the phrase doesn't make sense - are you missing the word "section"?

Reply:
it would appear I am